### PR TITLE
Tickets/dm 34878

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: python/lsst/ts/ess/common/device/mock_device.py [hunk 1]
```diff
@@ -33,6 +33,10 @@
 
 
 class MockDevice(BaseDevice):
+    # The wait time between sending telemetry (second). This can be adjusted by
+    # unit tests to mock connection timeouts.
+    wait_time = 1
```
Comment (r-owen) [inline python/lsst/ts/ess/common/device/mock_device.py:38]: I gently suggest something like "telemetry_interval" would be clearer.

File: doc/version_history.rst [hunk 2]
```diff
@@ -6,6 +6,19 @@
...
+v0.7.4
+======
+
+* Add wait_time class variable for mocking of timeouts.
+* Add pre-commit config file.
+* ``setup.cfg``: specify asyncio_mode=auto.
+
+Requires:
+
+* ts_tcpip 0.3
```
Comment (r-owen) [inline doc/version_history.rst:18]: Please consider 0.3.7 (which fixes monitoring) or 0.4 (not needed, just a rounder number)

**Review by r-owen — APPROVED** [review]
Looks fine. I had two minor optional suggestions.

File: python/lsst/ts/ess/common/device/mock_device.py (hunk 1 above)
Comment (wvreeven) [reply python/lsst/ts/ess/common/device/mock_device.py]: Done

File: doc/version_history.rst (hunk 2 above)
Comment (wvreeven) [reply doc/version_history.rst]: Done

**Pull request merged**